tools/license/get_dep_licenses.go

```diff
@@ -123,6 +123,9 @@ var (
 		"github.com/signalfx/com_signalfx_metrics_protobuf": true, // in manual_append
 		"github.com/bmizerany/assert":                       true, // has license in README.md
 		"github.com/kr/logfmt":                              true, // Readme mentions MIT license
+		"bitbucket.org/ww/goautoneg":                        true, // BSD-3-Clause in README
+		"github.com/munnerz/goautoneg":                      true, // BSD-3-Clause in README
+		"github.com/heketi/utils":                           true, // Apache 2.0 in README
```

Comment:
What are these new imports about?

Reply:
Kubernetes depends on these for whatever reason, so they are pulled in as transitive dependencies. I went to the repos and they have the licenses mentioned in the README -- not sure if that is the appropriate action here?